prisma/schema.prisma

```diff
@@ -134,8 +134,8 @@ model Block {
   time_since_last_block_ms Int?
   graffiti                 String             @db.VarChar
   size                     Int?
-  difficulty               BigInt
-  work                     BigInt?
+  difficulty               Decimal?
```

Comment:
I dont have strong preference on this, but it seems the difficulty type could be `Decimal` instead of nullable Decimal.